.github/workflows/ci-playwright.yml

```diff
@@ -0,0 +1,57 @@
+name: CI - Playwright
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - 'releases/*'
+
+jobs:
+  cross-browser-automation:
+    name: ${{ matrix.os }} Playwright@${{ matrix.playwright}} headless=${{ matrix.headless }}
+    strategy:
+      fail-fast: false
+
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-2019]
```

Comment:
Is there a 'windows-latest', if so I suggest using it so we don't fall out of date.

Reply:
There is window latest. but I put windows-2019 so we don't get windows 32 bit randomly. I believe window 2019 will give us 64 bit by default. I could be wrong.

Reply:
If we get Win32 randomly I think that's ok.  There are plenty of potential users that very well could be using this on Win32.